kinto/swagger.yaml

```diff
@@ -0,0 +1,2183 @@
+swagger: '2.0'
+
+externalDocs:
+  description: Full Documentation
+  url: http://kinto.readthedocs.io
+
+# May be redefined from settings
+info:
+  title: kinto
+  version: ''
+  description: >-
+    Kinto is a minimalist JSON storage service with synchronisation and sharing
+    abilities. It is meant to be easy to use and easy to self-host.
+
+
+    **Limitations of this OpenAPI specification:**
+      1. Validation on OR clauses is not supported (e.g. provide `data` or `permissions`
+      in patch operations).
+      2. [Filtering](http://kinto.readthedocs.io/en/stable/api/1.x/filtering.html)
+      is supported on any field by using `?{prefix}{field_name}={value}`. But swagger
+      doesn't have support for patterned querystrings.
+      3. [Backoff headers](http://kinto.readthedocs.io/en/stable/api/1.x/backoff.html)
+      may occur with any response, but they are only present if the server is under
+      in heavy load, so we cannot validate them on every request. They are listed only
+      on the default error message.
+      4. [Collection schemas](http://kinto.readthedocs.io/en/stable/api/1.x/collections.html#collection-json-schema)
+      can be provided when defining a collection and should be a valid JSON Schema object,
+      but this is not validated by this specification.
```

Comment:
I think we should unify this list of limitations with the ones in the documentation, and provide a link to them.